src/map/status.cpp

```diff
@@ -6027,6 +6027,18 @@ void status_calc_bl_main(struct block_list& bl, std::bitset<SCB_MAX> flag)
 #ifndef RENEWAL
 		status->matk_min = status_base_matk_min(status) + (sd?sd->bonus.ematk:0);
 		status->matk_max = status_base_matk_max(status) + (sd?sd->bonus.ematk:0);
+
+		if (sd != nullptr && sd->matk_rate != 100) {
+			status->matk_max = status->matk_max * sd->matk_rate/100;
+			status->matk_min = status->matk_min * sd->matk_rate/100;
+		}
+
+		if ((bl.type == BL_HOM && battle_config.hom_setting&HOMSET_SAME_MATK)  /// Hom Min Matk is always the same as Max Matk
+				|| (sc && sc->getSCE(SC_RECOGNIZEDSPELL)))
+			status->matk_min = status->matk_max;
```

Comment:
Please try to keep the code/formatting the same between renewal and prerenewal.
Suggest to copy lines 6126-6130 here.